k8s-daemonset/k8s/namerd.yml

```diff
@@ -0,0 +1,93 @@
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: namerd-config
+data:
+  config.yml: |-
+    admin:
+      port: 9990
+
+    namers:
+    - kind: io.l5d.k8s
+      experimental: true
+      host: localhost
+      port: 8001
+
+    storage:
+      kind: io.l5d.inMemory
+      namespaces:
+        outgoing: |
+          /srv        => /#/io.l5d.k8s/default/http;
+          /host       => /srv;
+          /tmp        => /srv;
```

Comment:
this has something to do with signaling deploys right?  I forget how that works...